steam_desktop_updater.py

```diff
@@ -185,8 +181,8 @@ def get_installed_apps(steam_root: Path):
     apps = []
     logging.info('Searching library folders')
     with (steam_root / 'steamapps' / 'libraryfolders.vdf').open('r') as lf:
-        library_folders = vdf.load(lf)['LibraryFolders']
-        for folder_path in [steam_root] + [Path(v) for k, v in library_folders.items() if k.isdigit()]:
+        library_folders = {k.lower(): v for k, v in vdf.load(lf).items()}["libraryfolders"]
+        for folder_path in [steam_root] + [Path(v["path"]) if isinstance(v, dict) else Path(v) for k, v in library_folders.items() if k.isdigit()]:
```

Comment:
Can you please split this into multiple lines? Maybe wrap the dict value in some `_getpath()` small function?